src/pages/Games.js

```diff
@@ -139,7 +140,7 @@ export class GameCard extends React.Component {
   getDate = () => {
     const { game, classes } = this.props;
     return <SubheadingDiv>
-        <FontAwesomeIcon icon='calendar-alt'/> {GamesStore.getDateString(game)} {GamesStore.getWeekDay(game)}
+    <FontAwesomeIcon icon='calendar-alt'/> { game.ended ? (GamesStore.getFullDateString(game) + ' ' + GamesStore.getWeekDay(game) + ' ' + game.time_start) : (GamesStore.getDateString(game) + ' ' + GamesStore.getWeekDay(game) + ' ' + game.time_start) }
```

Comment:
Do you think this whole "string" could be places in a separate formatting function? And fromatted with f-string maybe? 🤔 Then even maybe with a simple if-else statement.

Reply:
Possibly moving it to separate function would work but ifs didn't for some reason. Might be because I was doing something wrong but I've spent a lot of time trying to figure it out already...

Reply:
in a separate function it should work with if-else :)